client/my-sites/plans/main.jsx

```diff
@@ -69,11 +119,10 @@ const Plans = React.createClass( {
 } );
 
 export default connect(
-	( state, props ) => {
-		return {
-			plans: getPlans( state ),
-			sitePlans: getPlansBySite( state, props.sites.getSelectedSite() ),
-			siteId: getSelectedSiteId( state )
-		};
-	}
+	( state ) => ( {
+		plans: getPlans( state ),
+		sitePlans: getPlansBySite( state, getSelectedSiteId( state ) ),
```

Comment:
I think we provide redundant property `selectedSiteId` here. With this refactoring in place we could also get id directly from `selectedSite` object.

Though not quite sure if that is the recommended way to go, because it isn't the first time I see this redundancy. Personally I find this confusing to have two properties storing the same data.